react/Accordion/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from '../Icon'
+import styles from './styles.styl'
+
+class AccordionItem extends Component {
+  constructor(props) {
+    super(props)
+    this.onClick = this.onClick.bind(this)
```

Comment:
The [`cozy/cozy-guidelines`](https://github.com/cozy/cozy-guidelines#binding-event-handlers) says to avoid binding functions in the constructor.